src/codes.ts

```diff
@@ -9,11 +9,14 @@ class ErrorInvalidCode extends Error {
 }
 
 enum UnspentType {
+  p2pkh = 'p2pkh',
```

Comment:
I think this should be fine to add without breaking anything because it is not used in `codeList`.

Reply:
let's add p2wpkh as well then

Reply:
done